screens/Map.js

```diff
@@ -1,31 +1,92 @@
 import React from 'react';
 import {
-  Image,
-  Platform,
-  ScrollView,
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View,
-} from 'react-native';
-import { WebBrowser, MapView, Marker } from 'expo';
+  MapView,
+  Location,
+  Permissions,
+} from 'expo';
+
+import data from '../data.js'
 
 export default class Map extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      markers: [],
+      followsUserLocation: true,
+      mapRegion: {
+        latitude: 40.673868,
+        longitude: -73.970089,
+        latitudeDelta: 0.0922,
+        longitudeDelta: 0.0421,
+      },
+      locationResult: ""
+    };
+  };
+
   static navigationOptions = {
     header: null,
   };
 
+  componentDidMount = async () => {
+    this.setState({
+      markers: data
+    });
+    await this._getLocationAsync();
+  };
+
+  _getLocationAsync = async () => {
+    let {
+      status
+    } = await Permissions.askAsync(Permissions.LOCATION);
+    if (status !== 'granted') {
+      this.setState({
+        locationResult: 'Permission to access location was denied',
+        location,
+      });
+    }
+
+    let location = await Location.getCurrentPositionAsync({});
+ 
+    this.setState({
+      initialRegion: {
+        latitude: location.coords.latitude,
+        longitude: location.coords.longitude,
+        longitudeDelta: 0.005,
+        latitudeDelta: 0.005,
+      }
+    });
+  };
+
+  handlePress = (marker) => {
+    console.log('Placeholder for onPress triggered', marker)
+  };
+
+  regionChange = (event) => {
+    this.setState({
+      followsUserLocation: false,
```

Comment:
what does followsUserLocation do?

Reply:
I'm going to get rid of this on the new version that I'm pushing now. Previously, this was used to get the user's location but I'm not doing that in the componentDidMount so we dont need this anymore